projects/uitdatabank/reference/search.json

```diff
@@ -1010,6 +1022,15 @@
         "name": "q",
         "description": "An advanced query in Lucene syntax, allowing you to construct complex AND/OR filters on specific fields."
       },
+      "contributors":{
+        "schema": {
+          "type": "string",
+          "example": "technical-support@publiq.be"
```

Comment:
You can also add `"format": "email"` in the case of this parameter (inside `schema`)

Reply:
Thanks 👍